modules/ocf_mirrors/manifests/ftpsync.pp

```diff
@@ -42,11 +43,22 @@
     require => File[$project_path];
   }
 
-  cron { "ftpsync-${title}":
-    command => "BASEDIR=${project_path} ${project_path}/bin/ftpsync > /dev/null 2>&1",
-    user    => 'mirrors',
-    minute  => $cron_minute,
-    hour    => $cron_hour,
-    require => File["${project_path}/bin", "${project_path}/etc/ftpsync.conf"];
+  # TODO: Remove. Testing Only!
+  if $use_systemd {
+    ocf_mirrors::timer { "ftpsync-${title}":
+      exec_start => "/bin/bash -c 'BASEDIR=${project_path} ${project_path}/bin/ftpsync > /dev/null 2>&1'",
```

Comment:
Does this have to execute `bash` with a command like this? Can we just have the environment variable put into the systemd service file and then run it directly instead? I know that's supported with the `Environment` key in the unit file at least but I haven't tried that with timers specifically, but if it works it seems like it would be a bit cleaner.

Reply:
The ```exec_start``` is put in a service file. Part of the timer abstraction is that it creates both a timer and service file, and activates the service once the timer elapses. So, in short, yes we can do this-it just needs to be added to the template.

Reply:
Ok, yeah I think having an environment parameter that is passed to `ocf::systemd::service` would be good and could be used in other services. It would have to be an array or something, because you can specify multiple environment variables, but I don't think that would be too bad, it would just have to iterate through it in the template (erb is fun lol).